src/SDKs/SqlManagement/Management.Sql/Generated/SqlManagementClient.cs

```diff
@@ -565,33 +552,6 @@ public SqlManagementClient(ServiceClientCredentials credentials, params Delegati
             }
         }
 
-        /// <summary>
-        /// Initializes a new instance of the SqlManagementClient class.
-        /// </summary>
-        /// <param name='credentials'>
-        /// Required. Credentials needed for the client to connect to Azure.
-        /// </param>
-        /// <param name='httpClient'>
-        /// HttpClient to be used
-        /// </param>
-        /// <param name='disposeHttpClient'>
-        /// True: will dispose the provided httpClient on calling SqlManagementClient.Dispose(). False: will not dispose provided httpClient</param>
-        /// <exception cref="System.ArgumentNullException">
-        /// Thrown when a required parameter is null
-        /// </exception>
-        public SqlManagementClient(ServiceClientCredentials credentials, HttpClient httpClient, bool disposeHttpClient) : this(httpClient, disposeHttpClient)
```

Comment:
This is a breaking change, please bump version to `1.28.0-preview`

Reply:
Deepak, is this changed expected from latest autorest? Just want to check.

Reply:
Stefan, when you update csproj with new version, please also wipe out old release notes from 1.27.0 and add your new release notes.

Reply:
I'm not a 100% sure, but this is coming from AutoRest. @stefanb995 could you install latest autorest version and regen the code (`autorest --reset`)